plugins/completion/nvim-cmp/options/sources.nix

```diff
@@ -23,6 +23,10 @@ let
         options = {
           enable = mkEnableOption "";
           priority = intNullOption "";
+          entryFilter = mkOption {
```

Comment:
I have defined several custom functions to simplify the nix code for type definitions.
Please use the `strNullOption` function, which you can find in `helpers/custom_options.nix`.
This should exactly match your type definition